src/jit/hwintrinsicxarch.cpp

```diff
@@ -788,24 +788,12 @@ GenTree* Compiler::impHWIntrinsic(NamedIntrinsic        intrinsic,
         }
     }
 
-    if ((HWIntrinsicInfo::IsOneTypeGeneric(intrinsic) || HWIntrinsicInfo::IsTwoTypeGeneric(intrinsic)) &&
-        !HWIntrinsicInfo::HasSpecialImport(intrinsic))
+    if (HWIntrinsicInfo::IsOneTypeGeneric(intrinsic) && !HWIntrinsicInfo::HasSpecialImport(intrinsic))
```

Comment:
Now, we do not have any generic intrinsic in ISA classes and all the generic helpers are in vector classes. Perhaps, we no longer need the `HWIntrinsicInfo::IsOneTypeGeneric(intrinsic)`.

Reply:
I'll remove that in a follow-up PR, after confirming that all the generic functions have been finally removed (when I started on this PR, we still had some non-helper methods which were generic, but which I think have since been exploded).